tests/testthat/test-dm.R

```diff
@@ -344,6 +303,7 @@ test_that("subsetting for dm/zoomed_dm", {
 test_that("methods for dm/zoomed_dm work", {
   expect_identical(length(dm_for_filter), 6L)
   expect_identical(length(dm_zoom_to(dm_for_filter, t2)), 3L)
```

Comment:
```suggestion
  expect_identical(ncol(dm_zoom_to(dm_for_filter, t2)), 3L)
```

Reply:
sure, but we are testing the method `length.zoomed_dm()` here. So maybe just `skip_if_remote_src()`?

Reply:
Yes, and also add a FIXME to remind us that we need a test for `ncol()` if we don't have it yet.